qrisk/stats.py

```diff
@@ -314,10 +323,18 @@ def omega_ratio(returns, annual_return_threshhold=0.0):
 
     """
 
-    daily_return_thresh = pow(1 + annual_return_threshhold, 1 /
-                              APPROX_BDAYS_PER_YEAR) - 1
+    if returns.size < 2:
+        return np.nan
+
+    if annualization == 1:
+        return_threshold = required_return
+    elif required_return <= -1:
+        return np.nan
+    else:
+        return_threshold = (1 + required_return) ** \
+            float(1 / annualization) - 1
```

Comment:
nvm, we have the division future imported, the `float(...)` threw me off though, it doesn't do anything here so we should probably remove it.